plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/model/Role.js

```diff
@@ -23,7 +23,7 @@ Ext.define('NX.coreui.model.Role', {
     'id',
     'version',
     'source',
-    'name',
+    {name: 'name', sortType: 'asUCText'},
```

Comment:
minor style note, may be nicer to uniformlly use object config and not mix, /me shrugs